tests/tensorflow/hooks/test_session.py

```diff
@@ -45,7 +45,7 @@ def test_new_graph(out_dir):
 
 def test_uninit_sess_run(out_dir):
     train_op, X, Y = get_train_op_and_placeholders()
-    init = tf.compat.v1.global_variables_initializer()
```

Comment:
I see that some places you do tf.compat.v1 -> tf. and other places tf -> tf.compat.v1. 
Why?

Reply:
this particular occurrence is a miss. 
the tests/tensorflow files dont have compat because they are expected to be executed on TF 1.x containers. and tests/tensorflow2/ in TF 2.x containers.

Reply:
so tests/tensorflow wont work on TF2.x containers? 
How do we make sure that we have required coverage for TF2.x ?